upstream/CVE-2019-11253.json

```diff
@@ -0,0 +1,113 @@
+{
+  "id": "CVE-2019-11253",
+  "modified": "2019-09-27T16:53:31Z",
+  "published": "2019-09-27T16:53:31Z",
+  "summary": "Kubernetes API Server JSON/YAML parsing vulnerable to resource exhaustion attack",
+  "details": "Improper input validation in the Kubernetes API server in versions v1.0-1.12 and versions prior to v1.13.12, v1.14.8, v1.15.5, and v1.16.2 allows authorized users to send malicious YAML or JSON payloads, causing the API server to consume excessive CPU or memory, potentially crashing and becoming unavailable. Prior to v1.14.0, default RBAC policy authorized anonymous users to submit requests that could trigger this vulnerability. Clusters upgraded from a version prior to v1.14.0 keep the more permissive policy by default for backwards compatibility.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.1.0"
```

Comment:
```suggestion
              "introduced": "1.0.0"
```

Reply:
fixed in https://github.com/aquasecurity/vuln-list-k8s/pull/73/commits/d323c51598435d97390a447a9fe3dfba48388192